internal/cli/commons.go

```diff
@@ -56,6 +56,7 @@ func matchingAppsFinder(cmd *cobra.Command, args []string, toComplete string) ([
 		return nil, cobra.ShellCompDirectiveNoFileComp
 	}
 
+	app.API.DisableVersionWarning()
```

Comment:
There are lots of `ValidArgsFunction`s not declared by name, but as a anon function, i.e. `ValidArgsFunction: func(cmd *cobra.Command ... ) { ... }, ...`, directly in the Cmd structures

All of these seem to be missing. `commons.go` only contains the matchers used multiple times.

Reply:
Good catch! Fixed here: 8e5b5111